process-engine/process_engine_core
# 🐛 Use internal identity for process resumption

## Changes

Reassembling and resuming a ProcessInstance is now done with the ProcessEngine's internal token. 

It must be done this way, in case the owner of a ProcessInstance does not have the `can_read_process_model` claim. 
Otherwise, it would be impossible to retrieve the ProcessInstance, or the corresponding ProcessModel.

Such situations can occur, if an auth token expires, before the ProcessInstance is finished.

## Issues

PR: #324

## How to test the changes

- Start a ProcessInstance
- Let the auth token of the owner expire
- Restart the ProcessEngine
- See that the Process is resumed as expected
